style.css

```diff
@@ -98,10 +94,28 @@ background-color: white;
     z-index: 2;
     padding: 0px 15px 0px 0px;
 }
+.cart_list {
+    display: none;
+}
+.cart_list.active {
+    display: block;  
+    position: relative;
+    right: 50px;
+    z-index: 2;
+    padding: 20px 10px 10px 10px;
```

Comment:
You could shorten it to `20px 10px 10px`

Reply:
> You could shorten it to `20px 10px 10px`

changed it